src/rpdk/core/fragment/generator.py

```diff
@@ -75,11 +77,43 @@ def validate_fragments(self):
         self.__validate_no_transforms_present(raw_fragments)
         self.__validate_outputs(raw_fragments)
         self.__validate_mappings(raw_fragments)
+        lint_warnings = self.__validate_fragment_through_cfn_lint(raw_fragments)
+        if not lint_warnings:
+            LOG.warning("Module fragment is valid.")
+        else:
+            LOG.warning(
+                "Module fragment is valid, but there are warnings from cfn-lint "
```

Comment:
1) We don't really know the module fragment is valid yet, right?

2) Might be worth distinguishing terminology from one of [`cfn-lint`s rule levels](https://github.com/aws-cloudformation/cfn-python-lint/blob/master/docs/rules.md#rule-levels)?
```suggestion
                "Module fragment might be valid, but there are warnings from cfn-lint "
```

Reply:
will this only appear if the user running `validate` specifies increased verbosity?

Reply:
@PatMyron right. I'll edit both.

@jotompki no, for modules this runs all the time.